mantidproject/mantid
# System test for Pearl powder diffraction script

- [x] #15946 _PR is merged_

**Description of work.**

System test for the new pearl script, this script which is ran within this system test triggers the `pearl_focus`, which is the most commonly used command by the `Pearl` scientists according to Craig Bull.  I have created a similar folder structure to `\\britannic\Pearl\Mantid` within system test data. The `pearl_focus` will use existing calibrations files, which are usually generated once every start of cycle. Essentially another test case should be added within this system test soon which generated the calibrations files and uses it to focus.

**To test:**
- Run the test locally (remember to rerun cmake/build system tests to get the files) 
- Make sure all generated files are cleared upon completion of system test
- All builds pass



Fixes #15913
_Does not need to be in the release notes._



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
